src/main/webapp/sign-up.html

```diff
@@ -59,8 +59,16 @@ <h2>get notified with emails</h2>
                 <label for="unsubscribe">unsubscribe</label>
               </div>
             </div>
-            <input type="submit" value="submit">
+            <button id="form-submit" type="button">submit</button>
           </form>
+          <div id="subscribe-form-submitted">
+            <h1>form has been submitted!</h1>
+            <p>thanks for joining the noms fam! check your email for a subscription confirmation!</p>
+          </div>
+          <div id="unsubscribe-form-submitted">
+            <h1>form has been submitted!</h1>
+            <p>hope you have enjoyed your time with noms!</p>
+          </div>
```

Comment:
Nice, I like that you made different messages for unsubscribe/subscribe. One thing you could do is use the same modal but change the subtext on it!

Reply:
@areetaw had a question about this so adding clarification! 

So instead of having both `subscribe-form-submitted` and `unsubscribe-form-submitted` you can have just one <div> `form-submitted` and assign an id to the child <p> object (with no defined text). Then, on submit, show the modal but change the <p> text based on whether they unsubscribed/subscribed. That way instead of alternating between two modal objects, you use the same object and just change the text.

Reply:
Gotcha, makes sense!